extensions/internal-dictionary/src/index.tsx

```diff
@@ -0,0 +1,307 @@
+import {
+  Action,
+  ActionPanel,
+  Clipboard,
+  Form,
+  Icon,
+  List,
+  LocalStorage,
+  confirmAlert,
+  useNavigation,
+  showToast,
+  Toast,
+} from "@raycast/api";
+import { ReactNode, useEffect, useState } from "react";
+import Fuse from "fuse.js";
+import { createId } from "@paralleldrive/cuid2";
+import { z } from "zod";
+
+const DictionaryEntry = z
+  .object({
+    id: z.string().cuid2(),
+    title: z.string(),
+    description: z.string(),
+    score: z.number(),
+  })
+  .partial({ score: true });
+type DictionaryEntry = z.infer<typeof DictionaryEntry>;
+
+const DictionaryEntryList = DictionaryEntry.array();
+type DictionaryEntryList = z.infer<typeof DictionaryEntryList>;
+
+export default function Command() {
+  const [entries, setEntries] = useState<DictionaryEntry[] | null>(null);
+  const [results, setResults] = useState<DictionaryEntry[]>([]);
+  const [searchQuery, setSearchQuery] = useState<string>("");
+
+  const displayEntries = results.length > 0 ? results : entries;
+  if (entries == null) {
+    LocalStorage.allItems().then((storageEntries) => {
+      const parsedEntries: DictionaryEntry[] = Object.entries(storageEntries).map(([_, e]) => JSON.parse(e));
+      setEntries(parsedEntries);
+    });
+  }
+
+  const options: Fuse.IFuseOptions<DictionaryEntry> = {
+    includeScore: true,
+    keys: ["title", "description"],
+  };
+
+  function handleCreate(entry: DictionaryEntry) {
+    LocalStorage.setItem(entry.id, JSON.stringify(entry)).then(() => {
+      const newEntries = [...(entries || []), entry];
+      setEntries(newEntries);
+    });
+  }
+
+  function handleEdit(entry: DictionaryEntry) {
+    if (!entries) {
+      return;
+    }
+    const idx = entries.map((e) => e.id).indexOf(entry.id);
+    if (idx <= -1) {
+      return;
+    }
+    const newEntries = [...entries];
+    newEntries[idx] = entry;
+    LocalStorage.setItem(entry.id, JSON.stringify(entry)).then(() => setEntries(newEntries));
+  }
+
+  function handleDelete(entry: DictionaryEntry) {
+    if (!entries) {
+      return;
+    }
+    const idx = entries.map((e) => e.id).indexOf(entry.id);
+    if (idx <= -1) {
+      return;
+    }
+    confirmAlert({ title: "Delete " + entry.title + "?" }).then((confirmed) => {
```

Comment:
[confirmAlert](https://developers.raycast.com/api-reference/feedback/alert#alert.actionstyle) has a `Alert.ActionStyle` where you can set it to `Destructive`